walker.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""A Random Walk Simulation For Multiple Walkers"""
+
+# Python code for 2D random walk.
+# Source: https://www.geeksforgeeks.org/random-walk-implementation-python/
+import sys
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def create_walkers(number_of_steps, number_of_walkers):
+    """
+    Creates a list of numpy arrays which each hold the coordinates of a walker. The coordinates
+    are randomly chosen in a range from 0 to the number of steps
+    :param number_of_steps: the number of steps the walker will make. This defines the
+    dimension of the array which represents the walker
+    :param number_of_walkers: the number of walkers defines the number of arrays, which
+    will be created
+    :return: a list, the 'scene' which contains all walkers, so all arrays containing coordinates
+    """
+    scene = []
+    for _ in range(number_of_walkers):
+        # create the array of a walker and add it to the scene-list
+        walker = np.random.randint(low=0, high=number_of_steps,
+                                    size=(number_of_steps, 2))
+        scene.append(walker)
+
+    return scene
+
+  
```

Comment:
Would be a 10/10 with pylons if this whitespace was removed, so really good job with the styling!

Reply:
noo, missed that one... didn't notice it and pylint didn't tell me ;)